pdf/core/parser.go

```diff
@@ -54,9 +53,18 @@ type PdfParser struct {
 	streamLengthReferenceLookupInProgress map[int64]bool
 }
 
+type Version struct {
+	Major int
+	Minor int
+}
+
+func (v Version) String() string {
```

Comment:
https://github.com/unidoc/unidoc/wiki/UniDoc-Developer-Guide#12-doc-comments
// String returns the PDF version as a string.  Implements interface fmt.Stringer.